portal/src/graphql/portal/AuthenticationLoginIDSettings.tsx

```diff
@@ -46,16 +46,17 @@ const switchStyle = { root: { margin: "0" } };
 const WidgetHeader: React.FC<WidgetHeaderProps> = function (
   props: WidgetHeaderProps
 ) {
+  const { renderToString } = React.useContext(Context);
+
   return (
     <div className={styles.widgetHeader}>
       <Toggle
+        label={renderToString(props.titleId)}
```

Comment:
Can it accept `<FormattedMessage>`?

Reply:
Just found out that toggle label can accept JSX.Element, will update this